src/linter.js

```diff
@@ -9,6 +9,7 @@ import { terminalWidth } from 'cli';
 import * as constants from 'const';
 import { ARCH_DEFAULT, ARCH_JETPACK, CHROME_MANIFEST, INSTALL_RDF,
          MANIFEST_JSON } from 'const';
+import { BANNED_LIBRARIES, UNADVISED_LIBRARIES } from 'libraries';
```

Comment:
Yeah, I didn't know what to call it. They're like... "warning" or "caution"... I couldn't think of a better word 😞